src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -878,6 +881,14 @@ FullQName AdvertiserMinMdns::GetCommissioningTxtEntries(const CommissionAdvertis
             snprintf(txtPairingInstr, sizeof(txtPairingInstr), "PI=%s", *pairingInstruction);
             txtFields[numTxtFields++] = txtPairingInstr;
         }
+
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+        if (const auto & jointFabricMode = params.GetJointFabricMode(); jointFabricMode.has_value())
```

Comment:
So here you could do (if GetJointFabricMode returns `BitFlags<JointFabricMode>`):
```suggestion
        if (const auto & jointFabricMode = params.GetJointFabricMode(); jointFabricMode.HasAny())
```

and then print `jointFabricMode.Raw()` into the string.

Reply:
Addressed in the latest commit. Thanks for your feedback.